lib/src/js/README.md

```diff
@@ -0,0 +1,58 @@
+# JavaScript API
+
+This directory contains Dart Sass's implementation of the Sass JS API. Dart's JS
+interop support is primarily intended for _consuming_ JS libraries from Dart, so
+we have to jump through some hoops in order to effectively _produce_ a JS
+library with the desired API.
+
+JS support has its own dedicated entrypoint in [`../js.dart`]. The [`cli_pkg`
+package] ensures that when users load Dart Sass _as a library_, this entrypoint
+is run instead of the CLI entrypoint, but otherwise it's up to us to set up the
+library appropriately. To do so, we use JS interop to define an [`Exports`]
+class that is in practice implemented by a CommonJS-like[^1] `exports` object,
+and then assign various values to this object.
+
+[`../js.dart`]: ../js.dart
+[`cli_pkg` package]: https://google.com/google/dart_cli_pkg
```

Comment:
This URL is broken, should it be https://github.com/google/dart_cli_pkg#dart-cli-packager ?